### PR TITLE
docs: permissioned contract deployment with tooling

### LINKED ISSUE #917: update permissioned deployment to match dapp-agoric-basics

### LINKED ISSUE #931: Docs Revamp - Page 4
Page 4: 
Goal: Developer can deploy their own bundles to devnet and Emerynet

Should know:

- When to use which network
- What a proposal is
- How a proposal gets into production
- The workflow, what you need for what

Testing 
in ALL environments

Debugging
in ALL environments

### LINKED ISSUE #961: Docs Revamp - Permissioned Deployments - writeup
**User Story:** As a Developer I should be able to read the docs and understand how and where I can make permissioned deployments in the Agoric ecosystem. 

- Define a checklist that I can walk thru for Devnet / EmeryNet / Mainnet
- Define the process 
- Define all the commands / steps

### PR DESCRIPTION
**Pull request by dckc**
closes #917 
closes #961 
closes #931

rendered:

 - https://dc-deploy-contract.documentation-7tp.pages.dev/guides/coreeval/ - 4 pages
 - (note a few pages outside that section are also changed: getting started, agd cli)

TODO:

 - [x] detail on devnet deployment:
   - [x] make a community post about what you're up to before deploying to devnet. Be sure to choose a novel and appropriate name for your contract during this MN-2 phase.
   - [x] you're not expected to deploy example dapps as-is to devnet. They would collide in `agoricNames.instance` if everybody did that.

builds on some recent work in dapp-agoric-basics

 - https://github.com/Agoric/dapp-agoric-basics/pull/16
 - https://github.com/Agoric/dapp-agoric-basics/pull/21

### CODE REVIEW

File: main/guides/coreeval/local-testnet.md
Comment (dckc) [inline main/guides/coreeval/local-testnet.md:32]: this is a link to the `dc-vote-fee` branch, which is WIP

File: main/guides/coreeval/local-testnet.md
Comment (LuqiPan) [inline main/guides/coreeval/local-testnet.md:79]: _nitpick_

You probably meant to add another newline before the  tab below?

File: main/guides/coreeval/local-testnet.md
Comment (dckc) [reply main/guides/coreeval/local-testnet.md:32]: I plan to land `dc-vote-fee` first and change the link here before landing this one.

File: main/guides/coreeval/permissions.md
Comment (Chris-Hibbert) [inline main/guides/coreeval/permissions.md:7]:
```suggestion
And recursively, a property access `powers.P.Q.R` succeeds if
```

File: main/guides/coreeval/permissions.md
Comment (Chris-Hibbert) [inline main/guides/coreeval/permissions.md:55]: starting instances is widely accessible.

Comment (dckc) [reply main/guides/coreeval/permissions.md:55]: only if you have access to `zoe`, right?

I suppose every contract can do `zcf.getZoeService()`. hm.

**Comment by dckc** [comment]
@sufyaankhan note changes to the list of documented [bootstrap powers](https://dc-deploy-contract.documentation-7tp.pages.dev/guides/coreeval/permissions.html#selected-bootstrappowers); for example, `feeMintAccess` is no longer discussed.

The example is now from dapp-agoric-basics. It used to be an example of adding an Inter Protocol PSM for a new token. But we now have cosgov.org tooling for that sort of thing. So it no longer seems worthwhile to document Inter Protocol powers: `economicCommitteeCreatorFacet`, `econCharterKit`, `provisionPoolStartResult`, `psmKit`. Inter Protocol is like KREAd or Crabble in this way: we don't document the bootstrap powers involved in deploying those here; we leave that to https://community.agoric.com/c/governance/core-eval/31

**Review by kbennett2000 — APPROVED** [review]

**Pull request merged**
